Makefile

```
@@ -61,6 +61,23 @@ SWAGGER_GENERATE = rm -rf ./client; \
 		--tags="User" \
 		--tags="Cost Estimation"
 
+# E2E tests
+CY_API_URL           ?= http://172.42.0.3:3001
+# Env list specified in file /e2e/e2e.go
+CY_TEST_GIT_CR_URL   ?= git@172.42.0.14:/git-server/repos/backend-test-config-repo.git
+	
+# Local E2E tests
+# Note! Requires access to the private cycloid BE, only acessible within the organisation
+# AWS - ECR login 
+AWS_ACCESS_KEY_ID 	  ?= $(shell vault read -field=access_key secret/cycloid/aws) 
+AWS_SECRET_ACCESS_KEY ?= $(shell vault read -field=secret_key secret/cycloid/aws) 
+AWS_REGION            ?= eu-west-1
+AWS_ACCOUNT_ID        ?= $(shell vault read -field=account_id secret/cycloid/aws)
+# Local BE
+LOCAL_BE_GIT_PATH ?= /home/marcoldp/go/src/github.com/cycloidio/youdeploy-http-api#home/cycloidio/youdeploy-http-api
```

```suggestion
LOCAL_BE_GIT_PATH ?= /home/cycloidio/youdeploy-http-api
```

Or something like `../youdeploy-http-api` ?